server.js

```diff
@@ -100,9 +100,23 @@ module.exports = function(opts, callback) {
     });
   }
 
-  app.listen(process.env.PORT || opts.port, process.env.HOST || opts.bind, function() {
-    console.log("Listening at http://%s:%d/", this.address().address, this.address().port);
+  var handler = process.env.SOCKET || opts.socket || process.env.PORT || opts.port;
+  var server = app.listen(handler, process.env.HOST || opts.bind, function() {
+    var endpoint;
+    if (opts.socket) {
+      endpoint = opts.socket;
+      fs.chmodSync(opts.socket, '1766');
```

Comment:
Yep, it's `umask` to ensure that socket can be accessed from other users/groups, so you don't have to add nginx user, for example, to the group what runs tessera.